CardScan/Classes/TestingImageExtension.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  TestingImageExtension.swift
+//  CardScan
+//
+//  Created by Jaime Park on 12/3/19.
+//
+
+import Foundation
+
+extension ScanBaseViewController {    
```

Comment:
Can we extend the protocol instead? This method, logically, seems to go with the protocol. It makes sense to put it in an extension, if that works with swift (which I think it does)

Reply:
oooh this is what you meant by your message. got it!